paymentsheet/src/test/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowControllerTest.kt

```diff
@@ -2238,16 +2259,18 @@ internal class DefaultFlowControllerTest {
 
         flowController.confirm()
 
-        verify(launcher).launch(
-            eq(
-                CvcRecollectionData(
-                    lastFour = "4242",
-                    brand = CardBrand.Visa
-                )
-            ),
-            eq(PaymentSheet.Appearance()),
-            eq(false)
-        )
+        if (shouldRecollectCvc) {
```

Comment:
We should `verify(launcher, never()).launch(any()...)` for the else case.

Reply:
Updated